std_msgs/Header header
GraphObject[] objects